auth-tokens/src/main/java/com/palantir/tokens/auth/AuthHeader.java

```diff
@@ -27,14 +30,24 @@
 // see: https://jersey.java.net/apidocs/latest/jersey/javax/ws/rs/HeaderParam.html
 public abstract class AuthHeader {
 
+    // https://tools.ietf.org/html/rfc2617#section-1.2 for case insensitive auth-scheme
+    private static final String VALIDATION_PATTERN_STRING = "^bearer\\s+(.*)$";
```

Comment:
it may be faster to use http://docs.oracle.com/javase/7/docs/api/java/lang/String.html#regionMatches%28boolean,%20int,%20java.lang.String,%20int,%20int%29 instead of regex. Not sure it matters.